data.py

```diff
@@ -195,24 +197,33 @@ def load_query_string(self, data: str) -> None:
 
 class DataV3(Data):
     @staticmethod
-    def get_topics() -> List[topics.Topic]:
+    def get_topics(self) -> List[topics.Topic]:
+        # TODO: Not have to read configs everytime
         return [
             topics.WirelessModule.all().data,
             topics.Camera.overlay_message,
+            self.create_voltage_topic(),
             # TODO: BOOST currently publishes data in the deprecated V2 format
             # on the V3 topic. Uncomment below when updated.
             # topics.BOOST.recommended_sp,
             # topics.BOOST.predicted_max_speed,
```

Comment:
Unrelated to your PR, but can you uncomment these lines and remove the TODO?

This change was made in https://github.com/monash-human-power/BOOST/pull/63 and means we can see BOOST data on the camera system!